.travis.yml

```diff
@@ -5,5 +5,5 @@ before_install:
     - pod install --project-directory=Example
 script:
 - set -o pipefail
-- xcodebuild -workspace Example/DBGHTMLEntitiesExample.xcworkspace -scheme DBGHTMLEntitiesExample -sdk iphonesimulator -configuration Debug test -destination platform='iOS Simulator',OS='10.0',name='iPhone 7' | xcpretty -c
+- xcodebuild -workspace Example/DBGHTMLEntitiesExample.xcworkspace -scheme DBGHTMLEntitiesExample -sdk iphonesimulator -configuration Debug test -destination platform='iOS Simulator',OS='11.4',name='iPhone X' | xcpretty -c
```

Comment:
Modern travis doesn't require XCPretty.

Reply:
```
If your .travis.yml has a script key that looks like this:

script: set -o pipefail && xcodebuild -project Foo.xcodeproj -scheme Foo -destination 'platform=iOS Simulator,OS=11.3,name=iPhone X' build test | xcpretty
you can now use the default script by setting a few keys:

xcode_project: Foo.xcodeproj
xcode_scheme: Foo
xcode_destination: platform=iOS Simulator,OS=11.3,name=iPhone X
```
Default script does what you'd expect. be sure to remove the entire `script` block in the Yaml

Reply:
Example https://github.com/yahoo/YMCache/blob/master/.travis.yml#L2-L12

Reply:
Sounds good. Trying that now.